lib/bootloader.js

```diff
@@ -45,7 +45,12 @@
   // version without applicationCache intercepting us.
   function getShadowIndex(success, failure) {
     var req = new XMLHttpRequest();
-    req.open('GET', 'APPSHELL_PATHappshell.html', true);
+    var path = 'APPSHELL_PATHappshell';
+    var query = window.location.search.match(/revision=([^&#]+)/);
```

Comment:
We only want to be able to retrieve the currently activated revision of `appshell.html` (and `manifest.appcache`). When `?revision=` is used retrieving `index.html` the manifest will be stripped and the `appshell.html` dom swap wont happen. see https://github.com/ticketsolve/ember-cli-deploy-appshell/blob/824dfb9be76e210f7c4f5beb3f4ccb4e75208536/lib/bootloader.js#L197. Don't need to add the revision query here.